### ISSUE TITLE
As a role? I want to export reports  to excel format

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#46]

**Comment by lcabrini** [comment]
I think we either make a generic user story in which we say "as a user I want to export my reports to excel format" or we make several stories relating to different roles and different reports. The first option would be simpler, but would only be valid if you should be able to export *all* reports. Otherwise, we still have to create multiple user stories.

Then I wonder, is there any specific reason to limit to MS Excel format. I would probably have thought we should implement a generic export mechanism so that we can export to formats like: ODF, Excel, HTML, JSON, XML, etc.

Anyways, this isn't going to happen in this development cycle so we have some time to think about it.

**Comment by BorderITS** [comment]
Is this OK

**Comment by lcabrini** [comment]
Yes, that looks better. But we need to think more about the whole concept.

**Comment by BorderITS** [comment]
Sure
